launch/tier4_planning_launch/config/scenario_planning/common/motion_velocity_smoother/motion_velocity_smoother.param.yaml

```diff
@@ -9,13 +9,13 @@
 
     # curve parameters
     max_lateral_accel: 0.8             # max lateral acceleration limit [m/ss]
-    min_curve_velocity: 2.74           # min velocity at lateral acceleration limit [m/ss]
+    min_curve_velocity: 0.6            # min velocity at lateral acceleration limit and steering angle rate limit [m/s]
```

Comment:
Same here. Are there any valid reason for changing this value?

Reply:
Vehicles which have slower `steer_rate_limit` s can not handle the sharp turns with this `min_curve_velocity`

The value is about vehicle/curve specs and this value -which it is `0.5` currently- is ok for both sample veh. and ısuzu bus. Also for sample_map & gtu_map. We can update the param later if it will be needed.